### PR TITLE
imcdonald/de38770 dynamic width of eol activity links to account for mobile devices

### PR DESCRIPTION
**Pull request by imcdonald-d2l**
Used media query to dynamically set width of content so it is easily visible on a mobile device.
Demo page for testing added to `demo/d2l-sequence-navigator/d2l-sequences-content-eol-main.html`. You can open dev tools and set device to try various display widths.
[Original Rally Ticket](https://rally1.rallydev.com/#/289692574792d/detail/defect/387156470440)

Before:
before

After:
after

### CODE REVIEW

File: components/d2l-sequences-content-eol-main.js [hunk 1]
```diff
@@ -37,11 +37,14 @@ export class D2LSequencesContentEoLMain extends D2L.Polymer.Mixins.Sequences.Ret
 			.missed-count-text {
 				color: var(--d2l-color-celestine);
 			}
-			d2l-missed-activity{
+			d2l-missed-activity {
 				text-align: left;
 				width: 50%;
 				margin: auto!important;
 			}
+			@media (max-width:550px) {
```
Comment (ChrisKraljevicD2L) [inline components/d2l-sequences-content-eol-main.js:45]: nit: I'm a bit fuzzy on this, but I'm not sure if this is our mobile width breakpoint. Either [design.d2l](http://design.d2l) will have it listed somewhere in there, or you can reach out to Tara and ask :+1:

Comment (ianlaidlaw) [reply components/d2l-sequences-content-eol-main.js:45]: yeah, the mobile breakpoint for us (and I think standard) is 929px. Here's a screenshot from one of our design docs:
image

File: components/d2l-sequences-content-eol-main.js [hunk 2]
```diff
@@ -37,11 +37,14 @@ export class D2LSequencesContentEoLMain extends D2L.Polymer.Mixins.Sequences.Ret
...
+				d2l-missed-activity { width: 100%; }
```
Comment (ianlaidlaw) [inline components/d2l-sequences-content-eol-main.js:46]: very small nit: generally we've been spacing these out even if it's just a single item.
```
d2l-missed-activity {
  width: 100%;
}
```

File: components/d2l-sequences-content-eol-main.js (hunk 1 above)
Comment (imcdonald-d2l) [reply components/d2l-sequences-content-eol-main.js:45]: So if we set `929px` as breakpoint we get grossness below if the user's browser window is smaller. Not sure if we're ok with this?

also, what does `nit` stand for 😅 

smaller-desktop

File: components/d2l-sequences-content-eol-main.js (hunk 2 above)
Comment (imcdonald-d2l) [reply components/d2l-sequences-content-eol-main.js:46]: Thanks for letting me know 👍

**Pull request merged**